test/integ_tests/test_create_quantum_job.py

```diff
@@ -58,14 +58,16 @@ def test_failed_quantum_job(aws_session, capsys):
     # Check job is in failed state.
     assert job.state() == "FAILED"
 
+    timestamp = job.name.split("-")[-1]
```

Comment:
How will you get ts2, if we change that here?

Reply:
Ah fair enough. In that case, I'll just list keys for f"jobs/{job_name}/" and match against `\d+` for the timestamp in the asserts. Might as well make that change now instead of later

Reply:
Actually, that seems to lower the test coverage quite a bit since we could be validating assets from old jobs. I should be able to parse from metadata instead